pkg/logs/config/integration_config.go

```diff
@@ -43,6 +43,7 @@ type LogsConfig struct {
 	Source          string
 	SourceCategory  string
 	Tags            []string
+	DirectoryAsTag  bool `mapstructure:"directory_as_tag" json:"directory_as_tag"`
```

Comment:
Are you really sure this how we want to collect this tag ? It means that the user has to specifically assign the right value. I think that we would like something more seamless from a user perspective.

Reply:
You're right. Might make more sense to offer it by default (like most of the tags provided).
I'll change that and make sure it's enabled for wildcard specific paths.

Reply:
Updated. Now, the tag is added by default.